tests/test_vec_check_nan.py

```diff
@@ -68,3 +68,9 @@ def test_check_nan():
     else:
         assert False
 
+    try:
```

Comment:
Why put a try except if it should not fail?

Reply:
I think that It's normal data, I should be run successfully. Otherwise, it will make the test fail.

Reply:
you can remove the try/except then, if an exception occurs, the test will fail

Reply:
ok,  as you said